cypress/support/mockCompanies.js

```diff
@@ -0,0 +1,34 @@
+const mockCompanies = {
```

Comment:
Would mock companies be better off living in a fixture file? This works fine and seems to serve the same purpose, just curious on your reasoning on not making a new directory for all fixture files.